### ISSUE TITLE
Question searchbar system

### ISSUE THREAD

**Issue by Ventan00** [Ventan00/quanda#82]
We need to define how search system will search questions based on searchbar. 

- How sould we rank questions?
- Will there be suggested questions?
- How we could create suggestion system in searchbar

**Comment by pnarozniak** [comment]
What do you think about ranking questions based on upvotes on top answer. 
e.g.
Q-Question
A-Answer

> Q1:
>> A1 - 17+
>> A2 - 14+
>> A3- 10+

> Q2:
>> A1 - 33+
>> A2 - 11+

Search results:

1. Q2
2. Q1

**Comment by Ventan00** [comment]
Yes, we could include that, but I think keywords are a more important factor.
I would suggest prioritising keywords in headers over keywords in questions.
How about
10 * number of keywords in the header
5 * number of keywords in the question text
1 * upvotes for the top answer.
wdyt?

**Comment by pnarozniak** [comment]
I think that's well balanced.
